### PR TITLE
Impl content enricher sample

### LINKED ISSUE #3: Create a sample for Content Enricher
Create a sample for [Content Enricher](https://www.enterpriseintegrationpatterns.com/patterns/messaging/DataEnricher.html)

### PR DESCRIPTION
**Pull request by manuranga**
Fixes #3

### CODE REVIEW

**Review by ushirask — DISMISSED** [review]
LGTM.

File: content-enricher/content-enricher.bal [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import ballerina/http;
```
Comment (DimuthuMadushan) [inline content-enricher/content-enricher.bal:1]: Don't we need to add the license header?

Comment (manuranga) [reply content-enricher/content-enricher.bal:1]: Let's not do per file. We have a license for the repo.

**Review by DimuthuMadushan — APPROVED** [review]
LGTM

File: content-enricher/content-enricher.bal [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+type BankAccountReq record {|
```
Comment (manuranga) [inline content-enricher/content-enricher.bal:3]: Should be `BankAccountRequest`

**Pull request merged**